cleanup/artifactCleanup/artifactCleanup.properties

```diff
@@ -1,6 +1,8 @@
 // Support different policies for different repos. Parameters between '|' are optional
-// Policy syntax: [ <cron time>, [ <repo list> ], months |, <pause time>, <true|false dryrun> |, <true|false disablePropertiesSupport> | }
+// Policy syntax: [ <cron time>, [ <repo list> ], months |, <pause time>, <true|false dryrun> |, <true|false disablePropertiesSupport> |, <true|false> keepRelease |, releaseRegex | }
 // Example: [ "0 0 5 ? * 1", [ "libs-releases-local" ], 3, 500, true, true ],
 policies = [ 
                [ "0 0 5 ? * 1", [ "libs-releases-local" ], 3 ], 
            ]
+
+// regex = ~/.*-\d\.\d\.\d\.*/
```

Comment:
What's is the purpose of this regex defined out of a policy?

Reply:
It is used in line 57:
`def releaseRegex = params['releaseRegex'] ? config.regex as Pattern : ~/.*-\d\.\d\.\d\.*/`
 I was wondering if it would be convenient to have possibility to set it also outside of policy (like global setting for all repositories). Is it bad idea?

Reply:
I think we should keep all the configurations inside a policy to follow the current behavior. Otherwise we can question if we should have a global setting to all the other configurations, and I don't believe that is the main goal of your proposal.